### PR TITLE
Add hook for the bucket settings navgiation

### PR DESCRIPTION
**Pull request by ekala**
- The hook allows plugins to add navigations links on the settings
  navigation toolbar by introducing a new event: `swiftriver.bucket.settings.nav` Staging for #26
- We pass off an array with the `$bucket_base_url` and `$active` as the event data. This allows the plugins that hook into this event to access the URL of the bucket and also enables them manipulate the value of `$active`

### CODE REVIEW

File: themes/default/views/pages/user/layout.php [hunk 1]
```diff
@@ -34,7 +34,7 @@
 </nav>
 <?php endif; ?>
 
-<div id="content" class="user <?php echo $view_type ;?> cf">
```
Comment (69mb) [inline themes/default/views/pages/user/layout.php:37]: Why are you removing the user class?

Comment (ekala) [inline themes/default/views/pages/user/layout.php:37]: The user class distorts the alignment of the action buttons in the deployments listing (introduced by the Ushahidi push plugin) on the user dashboard i.e.

user Class Present

**With the class removed:**

user class removed

The visual impact on the other sections of the dashboard is a slight increase in the height of the settings containers

Comment (69mb) [inline themes/default/views/pages/user/layout.php:37]: This style is lost and the container titles have too much padding around them:

``` css
#content.user article.container div.property-title {
  padding: 5px 0;
}
```

Perhaps improve this one or add one to correct the alignment?

``` css
#content.user article.container a.remove-large {
  margin: 13px 10px 13px 0;
}
```

**Comment by ekala** [comment]
@69mb Made the css changes to:
- Preserve the padding around the container titles
- Accomodate having  action buttons in the container title section

**Comment by 69mb** [comment]
+1

**Pull request merged**